index.html

```diff
@@ -763,16 +763,15 @@ <h4 id="AB-TAG-constraints">2.5.1 Advisory Board and Technical Architecture Grou
           At the completion of the next regularly scheduled
           election for the TAG, the organization <em class="rfc2119">must</em> have returned to having at most
           one seat.</li>
-        <li>Two participants with the same primary affiliation per <a href="#AB-TAG-elections">2.5.2</a> must not both occupy elected seats on the AB.</li>
+        <li>Two participants with the same primary affiliation per <a href="#AB-TAG-elections">2.5.2</a> must not both occupy 		elected seats on the AB. If, for whatever reason, these constraints are not satisfied 
```

Comment:
Would you please remove the two tab characters in the middle of this line? Thank you!